barretenberg/ts/src/main.ts

```diff
@@ -102,7 +102,7 @@ async function initClientIVC(bytecodePath: string, crsPath: string) {
   debug('loading BN254 and Grumpkin crs...');
   // Plus 1 needed! (Move +1 into Crs?)
   const crs = await Crs.new(2 ** 18 + 1, crsPath);
-  const grumpkinCrs = await GrumpkinCrs.new(8192 + 1, crsPath);
+  const grumpkinCrs = await GrumpkinCrs.new(2 ** 14 + 1, crsPath);
```

Comment:
this was necessary after updating the fold_basic program from spawning 3 separate circuits to be folded to spawning 4. The size of the op queue is only ~800 as expected so I'm not sure why the ECCVM is so large. (I don't actually know how the number of ECCVM rows is expected to correspond to the size of `ultra_ops` but the number of `raw_ops` is `ultra_ops/2`). Perhaps this is just some quirk of the TS setup (which @lucasxia01 only just got working a few days ago) and not a reflection of the actual ECCVM size?